internal/pkg/skuba/addons/addons.go

```diff
@@ -275,6 +341,47 @@ func (addon Addon) Images(imageTag string) []string {
 	return images
 }
 
+func (addon Addon) listPatches() ([]string, error) {
+	result := []string{}
+	sourcePatches, err := filepath.Glob(filepath.Join(addon.patchResourcesDir(addon.addonDir()), "*.yaml"))
```

Comment:
Nitpicking: what if the user has files ending with ".yml" (it's a valid yaml extension)